templates/awsebscsidriverstorageclassdefault.yaml

```diff
@@ -6,9 +6,13 @@ metadata:
   namespace: kubeaddons
   labels:
     kubeaddons.mesosphere.com/provides: storageclass
+    kubeaddons.mesosphere.com/provides: defaultstorageclass
```

Comment:
can you have duplicate labels? Honestly asking here, never seen this before, should this be a list then?

Reply:
yeah i looked it upt, should be unique. So i am guessing this was a mistake.